x/ibc/03-connection/types/events.go

```diff
@@ -8,9 +8,10 @@ import (
 
 // IBC connection events
 const (
-	AttributeKeyConnectionID         = "connection_id"
-	AttributeKeyClientID             = "client_id"
-	AttributeKeyCounterpartyClientID = "counterparty_client_id"
+	AttributeKeyConnectionID            = "connection_id"
+	AttributeKeyClientID                = "client_id"
+	AttributeKeyCounterpartyClientID    = "counterparty_client_id"
+	AttributeKeyCounterpartyConnetionID = "counterparty_connection_id"
```

Comment:
typo in `AttributeKeyCounterpartyConnetionID` =>`AttributeKeyCounterpartyConnectionID`